site/src/pages/DeploySettingsPage/LicensesSettingsPage/LicensesSettingsPageView.tsx

```diff
@@ -55,13 +60,29 @@ const LicensesSettingsPageView: FC<Props> = ({
           description="Manage licenses to unlock Enterprise features."
         />
 
-        <Button
-          component={Link}
-          to="/deployment/licenses/add"
-          startIcon={<AddIcon />}
-        >
-          Add a license
-        </Button>
+        <Stack direction="row" spacing={2}>
+          <Button
+            component={Link}
+            to="/deployment/licenses/add"
+            startIcon={<AddIcon />}
+          >
+            Add a license
+          </Button>
+          <Tooltip title="Refresh license entitlements. This is done automatically every 10 minutes.">
+            <Button
+              onClick={() => {
+                if (refreshEntitlements) {
+                  if (refreshEntitlements()) {
```

Comment:
Why is this in an `if`?

Reply:
If it isn't `null`, I do not want to call a null function.

Reply:
Could you do `if (!!refreshEntitlements && refreshEntitlements())`